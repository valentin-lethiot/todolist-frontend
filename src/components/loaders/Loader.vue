<script setup lang="ts">
import { DotLottieVue } from '@lottiefiles/dotlottie-vue'
</script>

<template>
  <DotLottieVue
    style="height: 200px; width: 200px"
    autoplay
    loop
    src="https://lottie.host/be95a9f2-93a3-4f35-8279-fba6a3223b14/Bghk4QtCZ2.lottie"
  />
</template>
