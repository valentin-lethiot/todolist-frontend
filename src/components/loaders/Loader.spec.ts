import { describe, beforeEach, expect, test } from 'vitest'
import Loader from './Loader.vue'
import { mount, VueWrapper } from '@vue/test-utils'

describe('Loader', () => {
  let wrapper: VueWrapper

  beforeEach(() => {
    wrapper = mount(Loader)
  })

  describe('render', () => {
    test('should render the loader', () => {
      expect(wrapper.html()).toMatchSnapshot()
    })
  })
})
