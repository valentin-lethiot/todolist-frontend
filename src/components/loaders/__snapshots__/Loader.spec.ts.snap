// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Loader > render > should render the loader 1`] = `"<div style="height: 200px; width: 200px;"><canvas style="height: 100%; width: 100%;"></canvas></div>"`;
